### PR TITLE
configmap for issuers

### PR DESCRIPTION
**Pull request by chriskari**
**Description**

Changes proposed in this pull request:

- configmap for issuers

**Related issue(s)**
https://github.com/kyma-project/busola/issues/1957

### CODE REVIEW

File: extensions/configuration/dns-providers.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      spec.type.infoblox-dns: Infoblox DNS provider
+      spec.type.netlify-dns: Netlify DNS provider
+      spec.domains.include: Include Domains
+      spec.domains.exclude: Exclude Domains
+      included.domains: Included Domains
+      excluded.domains: Excluded Domains
+      domain.allowed: Domain that is allowed
+      domain.forbidden: Domain that is forbidden
+      spec.secretRef: Secret Reference
+      status: Status
```
Comment (mrCherry97) [inline extensions/configuration/dns-providers.yaml:138]: This shouldn't be here

File: tests/tests/namespace/test-api-rules.spec.js [hunk 2]
```diff
@@ -101,10 +101,13 @@ context('Test API Rules in the Function details view', () => {
...
-    cy.get('[aria-label="expand Required Scope"]:visible', { log: false })
-      .click();
+    cy.get('[aria-label="expand Required Scope"]:visible', {
+      log: false,
+    }).click();
 
-    cy.get('[data-testid="spec.rules.0.accessStrategies.0.config.required_scope.0"]:visible')
+    cy.get(
+      '[data-testid="spec.rules.0.accessStrategies.0.config.required_scope.0"]:visible',
+    )
```
Comment (mrCherry97) [inline tests/tests/namespace/test-api-rules.spec.js:110]: why this changes are here?

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: tests/tests/namespace/test-api-rules.spec.js (hunk 2 above)
Comment (chriskari) [reply tests/tests/namespace/test-api-rules.spec.js:110]: no clue haha

Comment (mrCherry97) [reply tests/tests/namespace/test-api-rules.spec.js:110]: as we have a talk, prettier things :D

File: extensions/configuration/issuers.yaml [hunk 3]
```diff
@@ -0,0 +1,187 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: issuers.cert.gardener.cloud
```
Comment (mrCherry97) [inline extensions/configuration/issuers.yaml:4]: Change to our naming convention

File: extensions/configuration/issuers.yaml [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: issuers.cert.gardener.cloud
```
Comment (mrCherry97) [inline extensions/configuration/issuers.yaml:7]: Change regarding to `metadata.name`

File: extensions/configuration/issuers.yaml [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+    busola.io/extension-version: '0.5'
+data:
+  details: |-
+    header:
+      - name: spec.acme.email
+        source: spec.acme.email
+      - source: 'status.state ? status.state : "UNKNOWN"'
+        highlights:
+          informative:
+          - UNKNOWN
```
Comment (mrCherry97) [inline extensions/configuration/issuers.yaml:18]: try to be consistent with indents - while yaml supports both, let's stick with an extra indent for lists

File: extensions/configuration/issuers.yaml [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+      {{[Issuer](https://cert-manager.io/docs/concepts/issuer/)}}
+      is a Namespace resource responsible for provisioning of certificates.
+  list: |-
+    - name: spec.acme.email
+      source: spec.acme.email
+    - name: status
+      source: 'status.state ? status.state : "UNKNOWN"'
+      highlights:
+        informative:
+        - UNKNOWN
```
Comment (mrCherry97) [inline extensions/configuration/issuers.yaml:132]: extra indent here as well, please

File: extensions/configuration/issuers.yaml [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+          widget: Table
+          showHeader: false
+          source: spec.acme.domains.exclude
+          children:
+            - source: $item
+              search: true
+  form: |-
+    - var: IssuerType
+      dynamicValue: | 
+        $exists(spec.acme) ? 'ACME' : $exists(spec.ca) ? 'CA' : ''
```
Comment (mrCherry97) [inline extensions/configuration/issuers.yaml:44]: probably it doesn't work, because if I provide VALID YAML to YAML section I can't create Issuer
Examples:
```
apiVersion: cert-manager.io/v1alpha2
kind: ClusterIssuer
metadata:
  name: letsencrypt-staging
spec:
  acme:
    # You must replace this email address with your own.
    # Let's Encrypt will use this to contact you about expiring
    # certificates, and issues related to your account.
    email: raspberry-pi@home.pi
    server: https://acme-staging-v02.api.letsencrypt.org/directory
    privateKeySecretRef:
      # Secret resource that will be used to store the account's private key.
      name: example-issuer-account-key
    # Add a single challenge solver, HTTP01 using nginx
    solvers:
    - http01:
        ingress:
          class: nginx
```
```
apiVersion: cert-manager.io/v1
kind: Issuer
metadata:
  name: ca-issuer
  namespace: mesh-system
spec:
  ca:
    secretName: ca-key-pair
```
check if it's related to this issues: 
https://github.com/kyma-project/kyma-dashboard/issues/51
https://github.com/kyma-project/busola/issues/2247

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: tests/tests/namespace/test-issuers.spec.js [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+
+    cy.get('[role=dialog]')
+      .contains('button', 'Create')
+      .click();
+
+    cy.url().should('match', new RegExp(`/issuers/${ISSUER_NAME}$`));
+  });
+
+  it('Inspect issuer', () => {
+    cy.contains(ISSUER_NAME);
```
Comment (mrCherry97) [inline tests/tests/namespace/test-issuers.spec.js:61]: we need to check values from create on details view, not only name
